### ISSUE TITLE
Test failure in Selenium/Agent/Admin/AdminPackageManager.t

### ISSUE THREAD

**Issue by bschmalhofer** [RotherOSS/otobo#1682]
This seems to reproducible when installing OTOBO packages from a local _.opm_ file. Installation seems to hand.

Here is the screenshot saved by Dev::UnitTest::Run.

![1163008993600000_1](https://user-images.githubusercontent.com/46019/159125027-0b5be507-75d8-4cd8-a176-447f25d671ab.png)

**Comment by svenoe** [comment]
I believe I have seen this, when cloud services or something like that was enabled, thus the package manager tried to verify the packages, but could not communicate with the verification server. Not 100% sure about it, though. If this is the reason, we maybe should print out a notification at least...

**Comment by bschmalhofer** [comment]
Tested this again and the the error didn't show up again. Tested both installation under Docker and Installation. 

Looking at the code I found that the package verify check is indeed calling home to https://portal.rother-oss.com/otobo/public.pl with a timeout of 120. A problem with `PackageVerify()` would also kind of explain the difference between installing and upgrading. In `Kernel::Modules::AdminPackageManager::_InstallHandling()` I can see a call to `PackageVerify()`. This call seem to be missing in `Kernel::Modules::AdminPackageManager::_UpdateHandling()`.
